src/ui/panel.ts

```diff
@@ -56,15 +58,19 @@ export class Panel {
   }
 
   async bridge(message: Message) {
-    switch (message.hide || message.login || message.manual_connection) {
-      case message.login:
-        // parsing email & start Imap eonnection
-        console.log(message);
+
+    switch (message !== undefined) {
+      // parsing email & start Imap eonnection
+      case isLogin(message):
+        const imapConfig: ImapConfig = emailParser(message as Login);
+
+        // It will alert the user using the manual connection if it can't find an email provider in the email providers list.
```

Comment:
I believe this will only be visible in the dev version when the console is open. For now that's fine but later would be good to show some error on the UI

Reply:
I used the log just for testing, but I replaced that log with the alert.